test/dummy/app/components/dummy/ui/share_component.slim

```diff
@@ -0,0 +1,27 @@
+div[
+  class=bem_class_name
+  class="d-ui-share"
+]
+  .d-ui-share__inner
+    - if @mobile_collapsible
+      = dummy_ui_icon(:share_2,
+                      height: 22,
+                      class_name: 'dropdown-toggle d-ui-share__toggle')
+
+    .d-ui-share__items
+      span.d-ui-share__title.small.font-weight-bold = t('.title')
+
+      - share_links.each do |link|
+        a.d-ui-share__item[
+          title=link[:title]
+          href=link[:url]
+          target=link[:target]
+          class="d-ui-share__item--#{link[:icon]}"
+        ]
+
+          = dummy_ui_icon(link[:icon],
+                          height: 22,
+                          class_name: 'd-ui-share__icon')
+      .d-ui-share__item
+        //TODO: change to new component
+        == cell('dummy/ui/clipboard', @url, height: 22)
```

Comment:
na desktopu ani mobilu se mi neukazuje, v mobile dropdownu je prazdne misto

Reply:
ikona doplněna, ale nefunguje ani to samotný copy (to má zkopírovat url stránky?). Dummy::Ui::ClipboardCell se bude přepisovat z cofee, tak nevím jestli to řešit, to už jí můžu udělat rovnou jako komponent taky...